packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -63,7 +64,7 @@ class MojoAssetBundle extends AssetBundle {
   }
 
   AssetBundleProxy _bundle;
-  Map<String, ImageResource> _imageCache = new Map<String, ImageResource>();
+  Map<String, ui.Image> _imageCache = new Map<String, ui.Image>();
```

Comment:
I don't think I understand this comment. We already deserialized the data coming from the mojo pipe into a single ui.Image object.